WolvenKit.RED4/Types/Primitives/Simples/ResourcePath.cs

```diff
@@ -10,6 +10,8 @@ namespace WolvenKit.RED4.Types;
 [DebuggerDisplay("{GetResolvedText()}", Type = "ResourcePath")]
 public readonly struct ResourcePath : IRedString, IRedPrimitive<string>, IEquatable<ResourcePath>, IComparable<ResourcePath>, IComparable
 {
+    public static readonly char DirectorySeparatorChar = '\\';
```

Comment:
Can't you get this from Path as default separator, or something like that?

Reply:
For Windows it would work, yes. But on Linux/Mac not.
CR2W ALWAYS uses `\` doesn't matter which OS